packages/storefront/src/pages/patterns/notifications/notifications.usage.md

```diff
@@ -71,16 +74,9 @@ the Decision Tree</p-link>
   Norman Group, 1994)
 - [Web Content Accessibility Guidelines](https://w3.org/WAI/standards-guidelines/wcag)
 
-<script lang="ts">
-import Vue from 'vue';
+<script lang="ts">import Vue from 'vue';
 import Component from 'vue-class-component';
-import {
-  BANNER_Z_INDEX,
-  FLYOUT_Z_INDEX,
-  MODAL_Z_INDEX, 
-  POPOVER_Z_INDEX,
-  TOAST_Z_INDEX
-} from '@porsche-design-system/components/src/constants';
+import {BANNER_Z_INDEX, FLYOUT_Z_INDEX, MODAL_Z_INDEX, POPOVER_Z_INDEX, TOAST_Z_INDEX} from '@porsche-design-system/components/src/constants';
```

Comment:
prettier stopped working?

Reply:
Prettier is enabled and working for me. My Webstorm settings did this and it's not wrong to me :)